coding/file_container.cpp

```diff
@@ -130,11 +129,9 @@ void FilesMappingContainer::Open(string const & fName)
   if (m_fd == -1)
     MYTHROW(Reader::OpenException, ("Can't open file:", fName));
 #endif
-
-  m_name = fName;
 }
 
-void FilesMappingContainer::Close()
+void MappedFile::Close()
 {
 #ifdef OMIM_OS_WINDOWS
   if (m_hMapping != INVALID_HANDLE_VALUE)
```

Comment:
Would be good to reset m_hMapping and m_hFile to INVALID_HANDLE_VALUE after CloseHandle